### DISCUSSION TITLE
How to disable stretch effect of notifications?

### DISCUSSION THREAD

**Discussion by shootheart** [hyprwm/Hyprland#7817] [Q&A]
Hi,
There is a stretch effect when multiple notifications popup(like video follows) but I just want to keep only fade in/out effect. How to disable it?

https://github.com/user-attachments/assets/5205fb4c-88fb-4d4d-a841-2b6079e0488b

**Comment by vaxerski** [comment]
disable animations for the layer

**Reply by shootheart** [reply]
Thanks for reply.
I have tried layer rule like `layerrule = noanim, notifications` to disable animations of layer, and there was no anymore animations. But what I want is keep ONLY fade in/out of it.

**Reply by vaxerski** [reply]
I don't think that's achievable atm.

**Comment by ghost** [comment]
You could potentially tweak this depending on your notification daemon too. Is that dunst?
